parachain/pallets/basic-channel/primitives/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "snowbridge-basic-channel-primitives"
```

Comment:
I'm wondering if this shouldn't go in `parachain/primitives` to match the other code's structure?